readthedocs/urls.py

```diff
@@ -34,6 +34,7 @@
     url(r'^support/error/$',
```

Comment:
I can move these to the new `support` module in this PR if that's ok.

Reply:
Seems reasonable 👍